### PR TITLE
Grid

### PR DESCRIPTION
**Pull request by AnuragSahu11**
Capture
image

*Grid of different categories and also added media queries for some of them,
*Added navbar 
*Changed some pixel units to rems.

### CODE REVIEW

File: components/grid.css [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+/* 2 col grid */
+
+/* 50 50 grid */
+
+.grid-2 {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+}
+.grid-2 > :nth-child(1) {
+  grid-column: 1/2;
```
Comment (junaidshaikh-js) [inline components/grid.css:12]: You don't need to specify the grid column for direct children as they automatically get that space.

**Review by junaidshaikh-js — COMMENTED** [review]
Good Work!

File: components/grid.css (hunk 1 above)
Comment (AnuragSahu11) [reply components/grid.css:12]: Got it , Thanks

File: components/grid.css [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+    display: grid;
+    grid-template-columns: 1fr;
+  }
+}
+
+/* navbar */
+
+.navbar {
+  background-color: white;
+  padding-left: 3%;
```
Comment (MaruthiPothuganti) [inline components/grid.css:75]: Suggestion: you can specify all paddings using the short-hand property.

**Review by MaruthiPothuganti — COMMENTED** [review]
Recommended changes: using Short-hand propert

File: components/grid.css (hunk 2 above)
Comment (AnuragSahu11) [reply components/grid.css:75]: Yes I will implement it right away

**Pull request merged**